packages/react-components/token-analyzer-preview/library/src/__tests__/sample-styles.ts

```diff
@@ -21,12 +21,14 @@ const useStyles = makeStyles({
 export const Component = () => {
   const styles = useStyles();
 
```

Comment:
Can be for later but let's add the focus indicator function calls here since we know we really want that to be accurate.